Client/Frontend/UserContent/UserScripts/Scripts_Dynamic/ScriptHandlers/Sandboxed/LoginsScriptHandler.swift

```diff
@@ -25,28 +25,26 @@ class LoginsHelper: TabContentScript {
     return profile.logins
   }
 
-  class func name() -> String {
-    return "LoginsHelper"
-  }
-
   required init(tab: Tab, profile: Profile, passwordAPI: BravePasswordAPI) {
     self.tab = tab
     self.profile = profile
     self.passwordAPI = passwordAPI
   }
 
-  func scriptMessageHandlerName() -> String? {
-    return "loginsManagerMessageHandler"
-  }
+  static let scriptName = "LoginsScript"
+  static let scriptId = UUID().uuidString
+  static let scriptSandbox: WKContentWorld = .defaultClient
+  static let messageHandlerName = "\(scriptName)_\(messageUUID)"
```

Comment:
More dynamic testing. These handlers:

```
webkit.messageHandlers.adsMediaReporting
webkit.messageHandlers.downloadManager
webkit.messageHandlers.loginsManagerMessageHandler
```

 don't exist in `.defaultClient` content world. I think the issue is that you changed the handler name here but not in the JS scripts.

Reply:
Fixed them all. They're sandboxed yeah, so I replaced the `$<message_handler>`. But I fixed those back just now, so now they're correctly injected for sandboxed scripts.

Only non-sandboxed scripts should have dynamic handlers :)